source/proof_address/lib/proof_address.rb

```diff
@@ -9,7 +9,7 @@
 module IdentityIdpFunctions
   class ProofAddress
     def self.handle(event:, context:, &callback_block)
-      params = JSON.parse(event['body'], symbolize_names: true)
+      params = event.transform_keys(&:to_sym)
```

Comment:
Do `symbolize_names` and `transform_keys` differ in how they handle nested maps?

Reply:
oh yikes good catch, it does not.

```
{ 'a' => { 'b' => 1 } }.transform_keys(&:to_sym)
# => {:a=>{"b"=>1}}
```

Gonna try plan B

Reply:
Plan B: 818b9411ebe3ee0f0196fb41d92a61b8ce040485